Gemfile

```diff
@@ -12,20 +15,20 @@ group :development do
 end
 
 group :development, :test do
-  gem 'yard', '~> 0.8'
-  gem 'yard-tomdoc'
   gem 'rake'
+  gem 'rubocop', '~> 0.50.0', require: false
   gem 'simplecov', require: false
-  gem 'rubocop', '~> 0.42.0', require: false
+  gem 'yard', '~> 0.8'
+  gem 'yard-tomdoc'
 end
 
 group :test do
-  gem 'futuroscope', github: 'codegram/futuroscope'
   gem 'danger-changelog', '~> 0.1'
+  gem 'futuroscope', github: 'codegram/futuroscope'
```

Comment:
Do we really need a reference into the HEAD of futuroscope? I suspect not and i would kill the whole git_source thing?

Reply:
Looks like the latest released `futuroscope` version on Rubygems is 0.1.11 from February 23, 2014. And there's [a bunch of commits](https://github.com/codegram/futuroscope/compare/v0.1.11...master) in `master` since then, including `Futuroscope::Pools::NoPool` implementation which is used in [hyperclient test suite](https://github.com/codegram/hyperclient/blob/d448da40f8a3a90c271b7c8db40a437597585fe1/test/test_helper.rb#L18).